src/main/webapp/script.js

```diff
@@ -11,3 +11,81 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+var jsonPlacehold = '{ "questions" : [' +
+'{ "q":"How would you spend an afternoon?" , "options":["hiking mountains", "shopping crafts", "trying local eats", "visiting a museum"]},' +
+'{ "q":"What would you choose for breakfast?" , "options":["acai bowl", "waffles", "crossaints", "huevos rancheros", "bagels"] },' +
+'{ "q":"Pick a TV show" , "options":["Emily in Paris", "Doctor Who", "Brooklyn 99", "Itaewon Class"] } ]}';
+var obj = JSON.parse(jsonPlacehold);
+var objIndex = 0;
+var responses = [];
+
+function onLoad(){
+    const quiz = document.getElementById('quiz');
+    quiz.appendChild(createParElement(obj.questions[0].q));
+    for (a in obj.questions[0].options){
+        // console.log(obj.questions[0].options[a])
+        var button = createButton(obj.questions[0].options[a])
+        button.addEventListener("click", function(){onClick(this);
+            });
+        quiz.appendChild(button);
+    }
+
+}
+
+function onClick(elm){
+    responses.push(elm.value);    
+    objIndex = objIndex+1;
+    // console.log(objIndex);
+    clearElm("quiz");
+    const quiz = document.getElementById('quiz');
+    quiz.appendChild(createParElement(obj.questions[objIndex].q));
+    for (a in obj.questions[objIndex].options){
+        // console.log(obj.questions[0].options[a])
+        var button = createButton(obj.questions[objIndex].options[a]);
+        if (objIndex==obj.questions.length-1){
+            button.addEventListener("click", function(){saveMatch(this);
+            });
+        }
+        else {
+            button.addEventListener("click", function(){onClick(this);
+            });
+        }
+        quiz.appendChild(button);
+    }
+}
+
+function saveMatch(elm){
+    responses.push(elm.value);
+    clearElm("quiz");
```

Comment:
"quiz" is used a lot in this file. I recommend putting it in a constant at the top so you only need to change it in one place if we want to:

`const QUIZ_ELEMENT_ID = "quiz";`